examples/tutorials/nb_python/Habitat_Interactive_Tasks.py

```diff
@@ -0,0 +1,2168 @@
+                dtype=torch.float,
+                device=self.device,
+            )
+
+            rewards = torch.tensor(
+                rewards, dtype=torch.float, device=self.device
+            ).unsqueeze(1)
+
+            current_episode_reward += rewards
+            next_episodes = self.envs.current_episodes()
+
+            # episode ended
+            if not_done_masks[0].item() == 0:
+                generate_video(
+                    video_option=self.config.VIDEO_OPTION,
+                    video_dir=self.config.VIDEO_DIR,
+                    images=rgb_frames[0],
+                    episode_id=current_episodes[0].episode_id,
+                    checkpoint_idx=0,
+                    metrics=self._extract_scalars_from_info(infos[0]),
+                    tb_writer=None,
+                )
+
+                print("Evaluation Finished.")
+                print("Success: {}".format(infos[0]["episode_success"]))
+                print("Reward: {}".format(current_episode_reward[0].item()))
+                print(
+                    "Distance To Goal: {}".format(
+                        infos[0]["object_to_goal_distance"]
+                    )
+                )
+
+                return
+
+            # episode continues
+            elif len(self.config.VIDEO_OPTION) > 0:
+                frame = observations_to_image(observations[0], infos[0])
+                rgb_frames[0].append(frame)
+
+        self.envs.close()
+
+
+# %%
+config_file = """
+BASE_TASK_CONFIG_PATH: "configs/tasks/rearrangement.yaml"
+TRAINER_NAME: "ddppo"
+ENV_NAME: "RearrangementRLEnv"
+SIMULATOR_GPU_ID: 0
+TORCH_GPU_ID: 0
+VIDEO_OPTION: ["disk"]
+TENSORBOARD_DIR: "data/tb"
+VIDEO_DIR: "data/videos"
+NUM_PROCESSES: 2
+SENSORS: ['RGB_SENSOR','DEPTH_SENSOR']
+CHECKPOINT_FOLDER: "data/checkpoints"
+NUM_UPDATES: 400
+LOG_INTERVAL: 10
+CHECKPOINT_INTERVAL: 50
+LOG_FILE: data/checkpoints/train.log
+EVAL:
+  SPLIT: "train"
+RL:
+  SUCCESS_REWARD: 2.5
+  SUCCESS_MEASURE: 'object_to_goal_distance'
+  REWARD_MEASURE: 'object_to_goal_distance'
+  GRIPPED_SUCCESS_REWARD: 2.5
+  PPO:
+    # ppo params
+    clip_param: 0.2
+    ppo_epoch: 2
+    num_mini_batch: 2
+    value_loss_coef: 0.5
+    entropy_coef: 0.01
+    lr: 2.5e-4
+    eps: 1e-5
+    max_grad_norm: 0.2
+    num_steps: 50
+    use_gae: True
+    gamma: 0.99
+    tau: 0.95
+    use_linear_clip_decay: False
+    use_linear_lr_decay: False
+    reward_window_size: 50
+    use_normalized_advantage: False
+    hidden_size: 256
+"""
+
+with open("configs/rl.yaml", "w") as f:
+    f.write(config_file)
+
+# %%
+# !rm -r data/tb
+
+import random
+
+import numpy as np
+import torch
+
+import habitat
+from habitat import Config, Env, RLEnv, VectorEnv, make_dataset
+from habitat.config import get_config
+from habitat_baselines.config.default import get_config as get_baseline_config
+
+# only run this if in colab environment
+if "google.colab" in sys.modules:
```

Comment:
No, we should definitely test this locally in the CI too... Why is this in the if statement like this. Unit testing this part is an important part of the script.

Reply:
But running the whole training process will take a while!

Reply:
@dexter1691 Can we short circuit locally? Like just run 1 or two iterations? And have that number be higher in the notebook?

Reply:
Yeah, I guess we can do that. 👍

Reply:
and in that case vut.is_notebook() might be a better way to check